main.nf

```
@@ -59,9 +68,55 @@ process transform_saige_base {
     """
     transform_base_saige.R ${saige_base}
     """
+    } 
+}
+
+
+
+/*---------------------------------
+  Transforming GWAS catalogue input 
+-----------------------------------*/
+
+if (params.gwas_cat_study_id) {
```

Now it's not obvious, but we only offer a certain set of study accession ids.

We need 1 check, to be able to fail early with an informative error message:
- check if the provided study accession is included in the list of study ids we have in our resource fail.
- if not inform the user that either the study accession id is invalid or it is not supported because it doesn't have harmonised summary stats.

Next week we need to implement 2 different checks to avoid being ambiguous. Adding as an issue.

---

Sounds good!